tests/run-macros/xml-interpolation-7/Macros_1.scala

```diff
@@ -0,0 +1,34 @@
+import scala.quoted._
+import scala.quoted.autolift
+
+import scala.language.implicitConversions
+
+case class Xml(parts: String, args: List[Any])
+
+object XmlQuote {
+
+  // Encoding for
+  //
+  // implicit class SCOps(s: StringContext) {
+  //   object xml {
+  //     def apply(exprs: Any*) = ...
+  //     def unapplySeq(...) = ...
+  //   }
+  // }
+  object XMLOps {
+    opaque type StringContext = scala.StringContext
+    def (ctx: scala.StringContext).xml: StringContext = ctx
```

Comment:
@liufengyun I added this other alternative that looks much simpler to use. Could you quickly check it and merge if it is ok.

Reply:
LGTM